tests/helpers/test-server.ts

```diff
@@ -166,10 +166,8 @@ class Server {
 }
 
 /** Returns a testing server */
-const createServer = (isHTTPS?: boolean) => {
-    const server = new Server(isHTTPS);
+export const createServer = (isHTTPS?: boolean) => {
+    const a = new Server(isHTTPS);
```

Comment:
Ups, I forgot to restore the name to `server`... but yes, maybe is better just do a return.